addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/handler/AutelisHandler.java

```diff
@@ -87,9 +90,29 @@
     private static int NORMAL_CLEARTIME = 60 * 60; // one hour
 
     /**
-     * Clear state after an command is sent
+     * Default poll rate rate, this is derived from the Autelis web UI
      */
-    private static int UPDATE_CLEARTIME = 60 * 2; // two minutes
+    static final int DEFAULT_REFRSH = 3; // 3 seconds
+
+    /**
+     * How long should we wait to poll after we send an update, derived from trial and error
+     */
+    static final int COMMAND_UPDATE_TIME = 6; // 6 seconds
```

Comment:
See above comment about the comment for the unit.

Reply:
yep